### PR TITLE
[ICDS-Dashboard] SDD added missing url parameter

### PR DESCRIPTION
**Pull request by lwyszomi**
Hi @Rohit25negi @calellowitz 

I fixed the issue in SDD report, so we didn't have a correct URL pattern and we didn't have one parameter. https://dimagi-dev.atlassian.net/browse/ICDS-717 

Need QA: no
Environment: ICDS
Locally tested: yes

Regards,
Łukasz

### CODE REVIEW

**Review by calellowitz — APPROVED** [review]
Seems fine but confirming this is thoroughly tested, given previous issues?

File: custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ function ServiceDeliveryDashboardController($scope, $http, $location, $routePara
 
     vm.dtOptions = DTOptionsBuilder.newOptions()
         .withOption('ajax', {
-            url: url('service_delivery_dashboard'),
+            url: url('service_delivery_dashboard', vm.step),
```
Comment (Rohit25negi) [inline custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js]: where do we actually set the value of this step variable?

**Review by Rohit25negi — APPROVED** [review]
this looks good to me. Asked One question. Also, can you please try testing it locally and see whether interaction with Backend is working fine with different filters?

File: custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js (hunk 1 above)
Comment (lwyszomi) [reply custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js]: we set this value 3 lines above  (line 19).

**Comment by lwyszomi** [comment]
@Rohit25negi @calellowitz I tested this with the backend and all works correctly

**Comment by Rohit25negi** [comment]
looks good to me. @calellowitz  `[ICDS]` looks fine to you?  of something like `[ICDS Dashboard]` you find better.

**Comment by lwyszomi** [comment]
@Rohit25negi @calellowitz title updated

**Comment by calellowitz** [comment]
I think including dashboard sounds better since there are lots of ICDS PRs that arent the dashboard. Thanks for changing @lwyszomi

**Pull request merged**